linkerd/app/gateway/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "linkerd2-app-gateway"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+http = "0.2"
+futures = "0.3"
+indexmap = "1.0"
+linkerd2-app-core = { path = "../core" }
+linkerd2-app-inbound = { path = "../inbound" }
+linkerd2-app-outbound = { path = "../outbound" }
+tokio = "0.2"
```

Let's make sure that this crate enables all the tokio feature flags it depends on. Otherwise, this crate won't compile if you cd into its directory and run `cargo build` --- it compiles when built from the workspace since other crates in the workspace enable its' features.

At a glance, it looks like the features this needs are:
```suggestion
tokio = { version = "0.2", features = ["rt-core", "macros"] }
```

Also, it looks like the Tokio dependency is only used for tests, so maybe it should be a dev-dep?